docs/UserGuide.md

```diff
@@ -513,28 +513,46 @@ Fig 5. Parts of an order
 
 Adds an order to the application.
 
-Format: `add-o [c/CLIENT_INDEX] [cn/CLIENT_NAME] [cp/CLIENT_PHONE] [ca/CLIENT_ADDRESS] [r/RECIPE_INDEX] [rn/RECIPE_NAME]
-[ri/INGREDIENT_NAME-QUANTITY-UNIT, ...] [op/ORDER_PRICE] od/ORDER_DEADLINE [oq/ORDER_QUANTITY]`
+All possible formats:
+1. All client and recipe details specified: <br>
+  `add-o cn/CLIENT_NAME cp/CLIENT_PHONE ca/CLIENT_ADDRESS
+  rn/RECIPE_NAME [ri/INGREDIENT_NAME-QUANTITY-UNIT, ...] op/ORDER_PRICE od/ORDER_DEADLINE [oq/ORDER_QUANTITY]`
+
+1. Only client index used (must provide recipe details): <br>
+  `add-o c/CLIENT_INDEX [cn/CLIENT_NAME] [cp/CLIENT_PHONE] [ca/CLIENT_ADDRESS]
+  rn/RECIPE_NAME [ri/INGREDIENT_NAME-QUANTITY-UNIT, ...] op/ORDER_PRICE od/ORDER_DEADLINE [oq/ORDER_QUANTITY]`
+
+1. Only recipe index used (must provide client details): <br>
+  `add-o cn/CLIENT_NAME cp/CLIENT_PHONE ca/CLIENT_ADDRESS r/RECIPE_INDEX [rn/RECIPE_NAME]
+  [ri/INGREDIENT_NAME-QUANTITY-UNIT, ...] [op/ORDER_PRICE] od/ORDER_DEADLINE [oq/ORDER_QUANTITY]`
+
+1. Both client and recipe indexes used: <br>
+  `add-o c/CLIENT_INDEX [cn/CLIENT_NAME] [cp/CLIENT_PHONE] [ca/CLIENT_ADDRESS] r/RECIPE_INDEX
+  [rn/RECIPE_NAME] [ri/INGREDIENT_NAME-QUANTITY-UNIT, ...] [op/ORDER_PRICE] od/ORDER_DEADLINE [oq/ORDER_QUANTITY]`
 
 <div markdown="block" class="alert alert-primary">
 
 **:bookmark: Note:**<br>
 
 * `od/ORDER_DEADLINE` includes date and time, it must follow the format specified [above](#4-features).
 
-Client details include client name, phone and address, they must be provided in one of these ways:
-* If `c/CLIENT_INDEX` is present, client details are copied from the client bookmark at the given index to the order.
-* If `cn/`, `cp/` or `ca/` are provided with `c/`, client details are taken from `cn/`, `cp/` or `ca/` instead.
-* If `c/` is not present, `cn/`, `cp/` and `ca/` must be provided.
-
-Recipe details include recipe name, ingredients and price. Recipe name and order price must be provided in one of these ways:
-* If `r/RECIPE_INDEX` is present, recipe details are copied from the recipe bookmark at the given index to the order.
-  `ORDER_PRICE` is calculated by multiplying the copied `RECIPE_PRICE` with `QUANTITY` of the order.
-* If `rn/`, `ri/` or `op/` is provided with `r/`, details are taken from `rn/`, `ri/` or `op/` instead.
-* If `r/` is not present, `rn/` and `op/` must be provided.
+**Client details** include client name, phone and address, they must be provided in one of these ways:
+* If `c/CLIENT_INDEX` is provided (Format 2 and 4),
+  * client details are copied from the client bookmark at the given index to the order.
+  * If `cn/`, `cp/` or `ca/` are provided with `c/`, client details are taken from `cn/`, `cp/` or `ca/` instead.
```

Comment:
If cn/, cp/ or ca/ is provided with c/, the respective client detail is taken from cn/, cp/ or ca/ instead.